src/chat/chatter.py

```diff
@@ -426,6 +426,11 @@ def send_the_orcs():
         # Aliases link
         menu_add("View Aliases", self.viewAliases, True)
 
+        # Show in Leaderboards
```

Comment:
I think comments are not needed here, the code documents itself via variable names.

Reply:
well, all others have one .. so ..